src/components/header.tsx

```diff
@@ -0,0 +1,226 @@
+import Image from "next/image";
+import * as React from "react";
+import Link from "next/link";
+import LanguageIcon from "@mui/icons-material/Language";
+import MenuIcon from "@mui/icons-material/Menu";
+import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
+import { Button } from "./ui/button";
+import { cn } from "@/lib/utils";
+import {
+  NavigationMenu,
+  NavigationMenuContent,
+  NavigationMenuItem,
+  NavigationMenuLink,
+  NavigationMenuList,
+  NavigationMenuTrigger,
+  navigationMenuTriggerStyle,
+} from "@/components/ui/navigation-menu";
+import { SheetTrigger, SheetContent, Sheet } from "@/components/ui/sheet";
+
+export default function Header() {
+  const hamburgerList = [
+    { text: "MZDY", link: "/wages" },
+    { text: "SPORENIE", link: "/savings" },
+    { text: "INVESÍCIE", link: "/investments" },
+    { text: "DANE", link: "/taxes" },
+    { text: "ČLÁNKY", link: "/news" },
+  ];
+  const headerMenuList = [
```

Comment:
Ďalšia vec čo mi napadla a bude sa ti hodiť do budúcnosti - ak máš nejaké takéto konštanty, ktoré nezávisia od stavu ani sa nijak počas behu programu nemenia, tak ich vyťahuj mimo komponent (kludne v rovnakom súbore, ale mimo komponent). Dôvod je ten, že v Reacte je každý komponent iba funkcia, ktorá sa zavolá na každý rerender. Takže ty na každý rerender vytváraš nové pole týchto linkov, aj keď je stále rovnaké a nič sa v ňom nemení.